server/app/routes/api/v1/annotations/index.js

```diff
@@ -1,16 +1,23 @@
 'use strict';
 var router = require('express').Router();
-module.exports = router;
+var _ = require('lodash');
+
+const sequelizeHandlers = require( 'sequelize-handlers' );
+const bluebird = require( 'bluebird' );
+const utils = require('../../../utils');
+const {ensureAuthenticated, ensureIsAdmin, Credentials, respondWith404, _err, db} = utils;
+const Annotation = db().models.annotation;
+const Resource = Annotation;
+
+/** see documentation at https://www.npmjs.com/package/sequelize-handlers */
 
-// router.use('/users', require('./users'));
-// router.use('/annotations', require('./annotations'));
-// router.use('/questions', require('.assessments/questions'));
-// router.use('/evaluations', require('.assessments/evaluations'));
-// router.use('/assessments', require('.assessments/assessments'));
+router.get(   '/',  sequelizeHandlers.query(Resource));
+router.get(   '/:id', ensureAuthenticated, sequelizeHandlers.get(Resource));
+router.post(  '/',  ensureAuthenticated,     sequelizeHandlers.create(Resource));
```

Comment:
I'm not sure if this is accurate, but the spacing looks strange on here. Which brings me to, don't forget to decide on a coding style together so that your project looks cohesive and not like 4 unique people worked on it! <-- as best as you can